### DISCUSSION TITLE
Segmentation fault(Core dumped) in -M Module

### DISCUSSION THREAD

**Discussion by sylyin0105** [neperfepx/neper#800] [Q&A] [answered]
@rquey Hello, I met a problem in M Module. My current Neper version is 4.8.0. However, when I change my version to 4.6.0，M Module works greatly with correct result. Here's my running log in version 4.8.0, and I'm waiting for your reply! Thank you.

neper -T -n 20 -id 200 -dim 3 -domain "cube(270,18,18)" -statcell id,x,y,z -statface id,x,y,z -o n20-id200-t1s1 -crysym cubic -orisampling uniform

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.8.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 128 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/public/home/xxx/.neperrc').
Info   : ---------------------------------------------------------------

Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n 20 -id 200 -dim 3 -domain cube(270,18,18)
         -statcell id,x,y,z -statface id,x,y,z -o n20-id200-t1s1 -crysym
         cubic -orisampling uniform
Info   : ---------------------------------------------------------------

Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :   - Setting seeds... 
Info   :   - Running tessellation...
Info   : Generating crystal orientations...
Info   :   - Crystal symmetry: cubic, no cut-off
Info   :   - Optimizing orientations (Thomson problem)...
Info   :     > Initial solution: f   =1.000000000
Info   :     > Iteration     40: fmin=0.000738320, f=0.000738320
Info   :     > Final solution  : f   =0.000738320 (40 iterations)
Info   : Writing results...
Info   :     [o] Writing file `n20-id200-t1s1.tess'...
Info   :     [o] Wrote file `n20-id200-t1s1.tess'.
Info   : Writing statistics...
Info   :     [o] Writing file `n20-id200-t1s1.stface'...
Info   :     [o] Wrote file `n20-id200-t1s1.stface'.
Info   :     [o] Writing file `n20-id200-t1s1.stcell'...
Info   :     [o] Wrote file `n20-id200-t1s1.stcell'.
Info   : Elapsed time: 4.420 secs.

neper -M n20-id200-t1s1.tess -format inp -interface cohesive -dim all

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.8.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 128 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/public/home/xxx/.neperrc').
Info   : ---------------------------------------------------------------

Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] n20-id200-t1s1.tess -format inp -interface cohesive
         -dim all
Info   : ---------------------------------------------------------------

Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `n20-id200-t1s1.tess'...
Info   :     [i] Parsed file `n20-id200-t1s1.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 8.177) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0.16|0.8/93%| 3%| 4%)  
Info   :   - Fixing 2D-mesh pinches...
Info   :   - 3D meshing... 100% (0.74|0.87/100%| 0%| 0%)
Info   :   - Processing mesh at interfaces...
Info   :     > Making mesh discontinuous...
Info   :     > Adding cohesive elements...
Info   :   - Switching mesh to order 2...
Segmentation fault (core dumped)

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
